### PR TITLE
Drilldown 

### PR DESCRIPTION
**Pull request by tosta1989**
how to add drill down to custom visual

### CODE REVIEW

**Comment by nikhilkabbin** [comment]
@tosta1989 How can I access drill level on update? Whether I am on level 1, 2 ?

File: Capabilities/Drilldown.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Drill down in a visualization
```
Comment (AviSander) [inline Capabilities/Drilldown.md]: in a Visual

File: Capabilities/Drilldown.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Visuals can now make use of PowerBI's Drill down support.
+(read more about [Drill down in a visualization in Power BI](https://powerbi.microsoft.com/en-us/documentation/powerbi-service-drill-down-in-a-visualization/))
```
Comment (AviSander) [inline Capabilities/Drilldown.md]: read more about it

File: Capabilities/Drilldown.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+
+## How to add drill down to your visual
```
Comment (AviSander) [inline Capabilities/Drilldown.md]: 'drill down support'

File: Capabilities/Drilldown.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+To add Drill down to your visual, add a new field to "capabilities.json" named `drilldown` who has one property:
+
+**roles** - the name of the dataRole you wanna enable to drill down. (the kind of the dataRole must be Grouping)
```
Comment (AviSander) [inline Capabilities/Drilldown.md]: you want to enable drill down on.

Comment (AviSander) [inline Capabilities/Drilldown.md]: Note: (the dataRole must be of 'Grouping' type)

File: Capabilities/Drilldown.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+Make sure that the property "max" in the conditions of the dataRole is set to 1. 
+(once you add this role to `drilldown`, you can drag multiple fields into the data role)
```
Comment (AviSander) [inline Capabilities/Drilldown.md]: no need for '('

Comment (AviSander) [inline Capabilities/Drilldown.md]: missing '.'

**Comment by tosta1989** [comment]
Hi @nikhilkabbin,
unfortunately there is no way inside update to know on what drill level you're on.

**Comment by nikhilkabbin** [comment]
Hi @tosta1989,

Thanks for the clarification. How do I use "Click to Turn on Drilldown" button in my code, can you point me to example? or give me a hint of how it works?

**Pull request merged**